### PR TITLE
Translated Community/index.html

### PR DESCRIPTION
**Pull request by khmai**
I've translated Community/index.html. Please help me to review it! Thanks

### CODE REVIEW

File: src/community/index.html [hunk 1]
```diff
@@ -6,121 +6,128 @@
...
-        <h1>Welcome to the Flutter community</h1>
+        <h1>Chào mừng đến với cộng đồng Flutter</h1>
         <p class="community__intro__description">
-            We're open source and would love to hear from you!
-            Below you'll find ways to get involved in the Flutter
-            community as well as links to
-            resources that can help answer your questions.
-            If you're in China, read
-            <a href="/community/china">Using Flutter in China</a>.
+            Chúng tôi là dự án mã nguồn mở và rất muốn nghe ý kiến từ bạn
```
Comment (truongsinh) [inline src/community/index.html]: missing "!" or "."

**Review by truongsinh — CHANGES_REQUESTED** [review]
Besides small typo, please revert all format-related changes. Please also noted that this is partially translated only, see https://deploy-preview-34--fluttervn-dev.netlify.app/community/ . It's fine to merge partially translated, but the format-related changes will make it harder later to keep up with the upstream.

You can see the changelog either with your own git tool or on https://github.com/truongsinh/website/pull/34/files

File: src/community/index.html (hunk 1 above)
Comment (khmai) [reply src/community/index.html]: Sorry, but which ones are format-related changes? :( Besides that shall I translate all the descriptions under certain links?

**Comment by truongsinh** [comment]
@KhanhMaiDang some example of formated change:

- https://github.com/truongsinh/website/pull/34/files#diff-75e72e45ed537716bfe468c9c0229af8L26-R58
- https://github.com/truongsinh/website/pull/34/files#diff-75e72e45ed537716bfe468c9c0229af8L73-R133

File: src/community/index.html (hunk 1 above)
Comment (truongsinh) [reply src/community/index.html]: see https://github.com/truongsinh/website/pull/34#issuecomment-623561962 for example of format-related changes. For this particular request, I mean it's missing "!" or "." and the end of the sentence.

**Review by ntvthuyen — COMMENTED** [review]
LGTM

File: src/community/index.html [hunk 2]
```diff
@@ -123,4 +122,4 @@ <h3>{{ case.name }}</h3>
             {% endif -%}
         {% endfor -%}
     </div>
-</section>
+</section>
```
Comment (truongsinh) [inline src/community/index.html:125]: Nit: this change is not needed, and might cause problem when merging w/ upstream later. But I'll let it pass for now.

File: src/community/index.html [hunk 3]
```diff
@@ -6,20 +6,19 @@
...
+            Ở bên dưới là cách để tham gia vào cộng đồng Flutter cũng như
+            những đường dẫn đến những nguồn có thể giúp bạn giải đáp những thắc mắc.
+            Nếu bạn ở Trung Quốc, hãy đọc
+            <a href="/community/china">Sử dụng Flutter ở Trung Quốc</a>.
         </p>
     </div>
 </section>
 
 {% comment %}
-    Featured services
```
Comment (truongsinh) [inline src/community/index.html:22]: Nit: this change is not needed, and might cause problem when merging w/ upstream later. But I'll let it pass for now.

**Pull request merged**